modules/developer_manual/pages/app/fundamentals/publishing.adoc

```diff
@@ -180,7 +181,8 @@ that:
 * major releases optionally featured on https://owncloud.org and sent to
 the owncloud-announce list.
 
-image:app/app-tile-verified.jpg[ownCloud "Verified" tag]
+.ownCloud "Verified" tag
+image:app/app-tile-verified.jpg[ownCloud 'Verified' tag]
 
 [[trusted]]
```

Comment:
same as above, maybe more ?

Reply:
I don't follow what you mean here. All I did was to change the about text and to add a caption; same for the image below it.